src/m365/teams/commands/meeting/meeting-create.ts

```diff
@@ -0,0 +1,206 @@
+          return `'${args.options.organizerEmail}' is not a valid email for organizerEmail.`;
+        }
+        return true;
+      }
+    );
+  }
+
+  /**
+   * Executes the command
+   * @param logger Logger instance
+   * @param args Command arguments
+   */
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const isAppOnlyAccessToken = accessToken.isAppOnlyAccessToken(auth.service.accessTokens[this.resource].accessToken)!;
+      if (isAppOnlyAccessToken && !args.options.organizerEmail) {
+        throw `The option 'organizerEmail' is required when creating a meeting using app only permissions`;
+      }
+      const graphBaseUrl = await this.getGraphBaseUrl(args.options);
+      const meeting = await this.createMeeting(logger, graphBaseUrl, args.options);
+      await logger.log(meeting);
+    }
+    catch (err: any) {
+      this.handleRejectedODataJsonPromise(err);
+    }
+  }
+
+  /**
+   * Gets the base MS Graph URL for the request
+   * @param options 
+   * @returns correct MS Graph URL for the request
+   */
+  private async getGraphBaseUrl(options: Options): Promise<string> {
+    let requestUrl = `${this.resource}/v1.0/`;
+    if (options.organizerEmail) {
+      const organizerId = await aadUser.getUserIdByEmail(options.organizerEmail);
+      requestUrl += `users/${organizerId}`;
+    }
+    else {
+      requestUrl += 'me';
+    }
+    return requestUrl;
+  }
+
+  /**
+   * Creates a new online meeting
+   * @param logger 
+   * @param graphBaseUrl 
+   * @param options 
+   * @returns MS Graph online meeting response
+   */
+  private async createMeeting(logger: Logger, graphBaseUrl: string, options: Options): Promise<any> {
+    if (this.verbose) {
+      logger.logToStderr(`Creation of a meeting...`);
+    }
+    const requestData: any = {};
+    if (options.participants) {
+      const attendees = options.participants.trim().toLowerCase().split(',').map(p => ({
+        upn: p.trim()
+      }));
+      requestData.participants = { attendees };
+    }
+    if (options.startTime) {
+      requestData.startDateTime = options.startTime;
+    }
+    if (options.endTime) {
+      requestData.endDateTime = options.endTime;
+    }
+    if (options.subject) {
+      requestData.subject = options.subject;
+    }
+    if (options.recordAutomatically !== undefined) {
+      requestData.recordAutomatically = options.recordAutomatically;
+    }
+    const requestOption: CliRequestOptions = {
+      headers: {
+        accept: 'application/json',
+        'content-type': 'application/json'
+      },
+      responseType: 'json',
+      method: 'POST',
+      url: `${graphBaseUrl}/onlineMeetings`,
+      data: requestData
+    };
+
+    try {
+      const requestResponse = await request.post<OnlineMeeting>(requestOption);
+      return requestResponse;
+    }
+    catch (error: any) {
+      if (error.response.status === 403) {
```

Comment:
Let's remove this check, try catch and error message. We need to implement something like this but at a different level for all graph commands. That's being discussed lately...

Reply:
Ok, removed.